### PR TITLE
Map constant return values of inlined calls.

### PR DESCRIPTION
**Pull request by ptersilie**

### CODE REVIEW

File: ykrt/src/compile/jitc_yk/trace_builder.rs [hunk 1]
```diff
@@ -465,15 +465,14 @@ impl<'a> TraceBuilder<'a> {
             for op in safepoint.lives.iter() {
                 match op {
                     aot_ir::Operand::LocalVariable(iid) => {
-                        live_args.push(self.local_map[iid]);
+                        live_args.push(self.local_map[iid].clone());
```
Comment (ptersilie) [inline ykrt/src/compile/jitc_yk/trace_builder.rs:468]: I realise I didn't need to make this change. I'm not sure we can even get constants here. Let me rectify this.

**Comment by ptersilie** [comment]
Fixed. Probably easier if I squash first before you review.

**Comment by vext01** [comment]
Go ahead.

**Comment by vext01** [comment]
LGTM

**Pull request merged**